### PR TITLE
Characterization Data

### PR DESCRIPTION
**Pull request by wm-c**
Another 20k lines of characterization data

### CODE REVIEW

File: characterization/characterization-project/build.gradle [hunk 1]
```diff
@@ -14,7 +14,7 @@ deploy {
             // or from command line. If not found an exception will be thrown.
             // You can use getTeamOrDefault(team) instead of getTeamNumber if you
             // want to store a team number in this file.
-            team = frc.getTeamOrDefault(5024)
+            team = frc.getTeamOrDefault(0)
```
Comment (ewpratten) [inline characterization/characterization-project/build.gradle]: Can you make the team number `5024` here?

File: characterization/characterization-project/src/main/java/dc/Robot.java [hunk 2]
```diff
@@ -93,7 +93,7 @@ public void robotInit() {
 
     // Note that the angle from the NavX and all implementors of wpilib Gyro
     // must be negated because getAngle returns a clockwise positive angle
-    AHRS navx = new AHRS(SPI.Port.kMXP);
+    AHRS navx = new AHRS("SPI.Port.kMXP");
```
Comment (ewpratten) [inline characterization/characterization-project/src/main/java/dc/Robot.java]: Remove the quotes here

**Review by ewpratten — CHANGES_REQUESTED** [review]

**Comment by ewpratten** [comment]
Can you also flip the right encoder in RobotConstants to match the characterization config?

**Comment by wm-c** [comment]
@Ewpratten 
Didn't see your message, will do

**Comment by ewpratten** [comment]
There is a chance this merge will mess up the simulator.. We shall find out

**Pull request merged**